src/System.Globalization/tests/Common/Unicode_5_1/PropList.txt

```diff
@@ -0,0 +1,1207 @@
+# PropList-5.1.0.txt
+# Date: 2008-03-20, 17:55:27 GMT [MD]
+#
+# Unicode Character Database
+# Copyright (c) 1991-2008 Unicode, Inc.
+# For terms of use, see http://www.unicode.org/terms_of_use.html
```

Comment:
@stephentoub @ellismg  : This file just lists the Unicode characters. This test (CharUnicode and CharNumericCategory) seem to be using Unicode6.3 though. Do I raise an issue to update the embedded resource UnicodeData.txt to represent Unicode 8?
I do not know the answer to this question regarding the copyright info. It seems to be the readme.txt from http://www.unicode.org/Public/UCD/latest/ReadMe.txt